rbvmomi.gemspec

```diff
@@ -16,7 +16,7 @@ Gem::Specification.new do |spec|
   spec.license  = 'MIT'
 
   spec.bindir = 'exe'
-  spec.files  = `git ls-files -z`.split("\x0").reject { |f| f.match(/^test\//) }
+  spec.files  = %w{LICENSE README.md} + Dir.glob("{lib,exec}/**/*")
```

Comment:
I think you meant to put `exe` rather than `exec` in there, but that isn't really needed as you'll never require it and it's pulled into the gem through the `spec.executables` line below. What _is_ needed, however, is the `vmodl.db` file in the root of the repository. Other than that, I think this looks good.

Reply:
Oops. Nice catch. I pushed a few for the right path.